src/search.js

```diff
@@ -39,7 +39,13 @@ export default class Search {
    * Create Search instance
    * @param {Core} core Core reference
    */
-  constructor(core) {
+  constructor(core, options) {
+  
+  
+    const providedAdapters = options.adapters || [];
+    const useAdapters = [VFSSearchAdapter, ...providedAdapters];
```

Comment:
The `VFSSearchAdapter` adapter needs to be written.

Reply:
@andersevenrud 
you said that only MySearchAdapter should be define not VFSSearchAdapter

Reply:
This is why I said that my code was only for illustration purposes.

There was more changes needed, but I didn't write it because I explained what was needed.